src/translations/pt/core.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  heroTitle: "Construa sistemas de design melhores",
+  heroSubtitle:
+    "Uma lista de verificação de código aberto para ajudá-lo a planejar, construir e expandir seu sistema de design.",
+  heroAction: "Comece agora",
+  
+  more: "Precisa de mais?",
+  about: "Sobre",
+  share: "Compartilhar",
+  contribute: "Contribuir",
+  
+  previous: "Anterior",
+  next: "Próximo",
+  
+  exportTitle: "Divulgue",
+  exportSubtitle:
+    "Os sistemas de design são melhores construídos em conjunto. É por isso que fizemos ser possível compartilhar esta lista de verificação, incluindo os itens que você já verificou. Compartilhe seu progresso com sua equipe para que todos possam acompanhar juntos.",
```

Comment:
My suggestion for the second sentence would be: "É por isso que tornamos possível compartilhar esta checklist...". 
In this case, the verb "fizemos", makes the sentence a bit wierd when it clashes with the "ser" verb.